template/api.mustache

```diff
@@ -25,21 +25,21 @@ namespace {{packageName}}.{{apiPackage}}
                 /// {{summary}}
                 /// </summary>
             /// <remarks>
-                /// {{notes}}
+                /// {{{notes}}}
```

Comment:
Why do we need three "{"? 😁

Reply:
To correctly render backtick symbol (&#x60;) without replacing it to `&#x60;`